src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,23 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { FC } from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo:FC<Props> = ({ todo }) => {
+  const className = `TodoInfo ${
+    todo.completed
+      ? 'TodoInfo--completed'
+      : ''}`;
+
+  return (
+    <article className={className}>
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      {todo.user
+        ? <UserInfo user={todo.user} />
+        : ''}
+    </article>
+  );
+};
```

Comment:
1. Let's use destructing assigment for todo here.
2. Why are you returning emprty string if user doesn't exist ? Let's change this logic to `AND` operator and don't render anything if user doesn't exist.

```suggestion
export const TodoInfo:FC<Props> = ({ todo }) => {
  const className = `TodoInfo ${
    todo.completed
      ? 'TodoInfo--completed'
      : ''}`;

  return (
    <article className={className}>
      <h2 className="TodoInfo__title">{todo.title}</h2>
      {todo.user && (
        <UserInfo user={todo.user} />
       )}
    </article>
  );
};
```